src/form/bf_mx_lin.jl

```diff
@@ -462,6 +464,43 @@ function constraint_mc_load_power(pm::LPUBFDiagModel, load_id::Int; nw::Int=nw_i
 end
 
 
+"""
+    constraint_mc_generator_power_delta(pm::LPUBFDiagModel, nw::Int, gen_id::Int, bus_id::Int, connections::Vector{Int}, pmin::Vector{<:Real}, pmax::Vector{<:Real}, qmin::Vector{<:Real}, qmax::Vector{<:Real}; report::Bool=true, bounded::Bool=true)
+
+Delta-connected generators similar to delta load model
+"""
```

Comment:
Can you add a LaTeX Math representation to the docstring? Note that you will need to change the string to 

```julia
@doc raw"""
...
"""
```

to allow for escaping necessary for math